bindata/deployment/prometheus-operator/prometheus-operator.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: metallb-controller
+  namespace: '{{.NameSpace}}'
+spec:
+  selector:
+    matchLabels:
+      component: controller
+  namespaceSelector:
+    matchNames:
+      - metallb-system
```

Comment:
do u need to render Namespace here and below so it will work on d/s ?

Reply:
:+1:

Reply:
I'd reword this to "if the user chooses to install into another namespace"

Reply:
done